### PR TITLE
Fix SegmentedButtonGroup with corner rounded

### PR DESCRIPTION
**Pull request by joshwooding**
Was a quick fix.

 @origami-z how are we handling changesets for stuff like this?

### CODE REVIEW

**Comment by origami-z** [comment]
> @origami-z how are we handling changesets for stuff like this?

I think we treat it as bug fix, but specially calling it only impacting theme next / rounded corner

**Review by origami-z — DISMISSED** [review]
Do we want to snapshot this iin theme next if fixed?

**Comment by joshwooding** [comment]
> Do we want to snapshot this iin theme next if fixed?

I'm not sure why it's not getting picked up, I guess because the kitchen sink imports other stories

**Comment by origami-z** [comment]
Kitchen sink is not included in snapshot.

I mean this: https://github.com/jpmorganchase/salt-ds/blob/c36e8a4466bbfd15e3ad152a37056b1f4cb8e522/packages/core/stories/text/text.qa.stories.tsx#L100

**Review by origami-z — APPROVED** [review]

**Review by bhoppers2008 — DISMISSED** [review]

File: packages/core/stories/segmented-button-group/segmented-button-group.qa.stories.tsx [hunk 1]
```diff
@@ -38,7 +38,20 @@ export const Default: StoryFn<SegmentedButtonGroupProps> = () => {
...
 
 Default.parameters = {
-  chromatic: { disableSnapshot: false },
+  chromatic: {
+    disableSnapshot: false,
+    modes: {
+      theme: {
+        themeNext: "disabled",
+      },
+      themeNext: {
```
Comment (origami-z) [inline packages/core/stories/segmented-button-group/segmented-button-group.qa.stories.tsx:47]: Now this is more obvious, I'll need to look into why this is not taken into effect

Comment (joshwooding) [reply packages/core/stories/segmented-button-group/segmented-button-group.qa.stories.tsx:47]: Fixed it

**Review by origami-z — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**